### PR TITLE
Add doctrine2 to ZendExpressive Module

### PR DESCRIPTION
**Pull request by Rud5G**
Codeception\Module\ZendExpressive implements DoctrineProvider

### CODE REVIEW

**Comment by Naktibalda** [comment]
Thanks,

Does it work when multiple requests are performed in a single test?
Other modules supporting Doctrine2 ([Symfony ](https://github.com/Codeception/Codeception/blob/2.3.5/src/Codeception/Lib/Connector/Symfony.php#L66-L80)and [ZF2](https://github.com/Codeception/Codeception/blob/2.3/src/Codeception/Lib/Connector/ZF2.php#L179-L181) with ServiceManager 2.*) implement service persistence by injecting services from the first instance of service manager to new instances created for later requests.

**Comment by Rud5G** [comment]
@Naktibalda i will take a look into this. 
Are there any tests at the moment in the repository for this? i couldn't find them on the first glance.

**Comment by Naktibalda** [comment]
Tests of this module are in https://github.com/Naktibalda/codeception-zend-expressive-tests
You can copy Doctrine tests from https://github.com/Naktibalda/codeception-zf2-tests/tree/2.2/tests/doctrine

File: src/Codeception/Module/ZendExpressive.php [hunk 1]
```diff
@@ -48,8 +50,46 @@ class ZendExpressive extends Framework
      */
     public $application;
 
+
     protected $responseCollector;
 
+    /**
+     * Grabs a service from Interop container.
+     * Recommended to use for unit testing.
```
Comment (DavertMik) [inline src/Codeception/Module/ZendExpressive.php]: ...for integration testing

**Comment by DavertMik** [comment]
Looks good to me. If this works for you, @Rud5G I think we can merge it 
Yes, better to have tests for Doctrine but It requires changes for external repositories so we can apply that later

**Comment by Naktibalda** [comment]
It looks good, but it probably doesn't work.

**Pull request closed without merging**